.circleci/config.yml

```diff
@@ -237,6 +237,51 @@ jobs:
       - store_test_results:
           path: ./cucumber/results
 
+  test-e2e-phase2-core:
```

Comment:
Could we refactor these jobs and use [parameterised jobs](https://circleci.com/docs/reusing-config/#authoring-parameterized-jobs)?

Just thinking we now have 4 CircleCI job that are all very similar with the only differences being (from my quick scan): `MESSAGE_ENTRY_POINT`, `PHASE2_CORE_CANARY_RATIO` and whether to use `NEXTUI`.